Delete old build files

hchoura opened the pull request.

Commit:
Delete old build files

jlhughes commented on src/gen.sh:
```diff
@@ -2,5 +2,9 @@
 cd "$SRCDIR"
+
+# Delete old build files
+rm -r ../build/
```
does 'make clean' not do this? or you want to run it each time?

hchoura replied:
No. 
make clean do:
rm -f build/foam2.jar
rm -f build/foam2.classpath.txt
rm -rf .foam2-gensrcs

jlhughes replied:
so why not add this to make clean?

jlhughes replied:
with the clean change above, this is no longer needed correct?

Commit:
Delete all the files in the build

Delete all files (classes, jar).
To be able to work on a fresh copy.
I would like to keep both ways (for cmd & ide).

jlhughes commented:
Superceded by https://github.com/foam-framework/foam2/pull/2639
